public/blog/2016-08-making-recommendations/making-recommendations.md

```diff
@@ -0,0 +1,278 @@
+
+How can I know what you may like if I have almost no information about you? The answer: taking [Collaborative filtering (CF)](https://en.wikipedia.org/wiki/Collaborative_filtering) approaches. Basically, this means to combine all the little knowledge we have about users and/or items in order to build a grid of knowledge with which we make recommendation.
+
+To help you with that, Biolab has written [Orange3-Recommendation](https://github.com/biolab/orange3-recommendation) - an add-on for [Orange3](http://orange.biolab.si/download/) to train recommendation models, cross-validate them and make predictions.
+
+
+### 
+
+
+
+
+### Input data
+
+
+First things first. Orange3-Recommendation can read files in native tab-delimited format, or can load data from any of the major standard spreadsheet file type, like CSV and Excel. Native format starts with a header row with feature (column) names. Second header row gives the attribute type, which can be continuous, discrete, string or time. The third header line contains meta information to identify dependent features (class), irrelevant features (ignore) or meta features (meta).
+
+Here are the first few lines from a data set:
+
+    
+        tid      user        movie       score
+        string   discrete    discrete    continuous
+        meta     row=1       col=1       class
+        1        Breza       HarrySally  2
+        2        Dana        Cvetje      5
+        3        Cene        Prometheus  5
+        4        Ksenija     HarrySally  4
+        5        Albert      Matrix      4
+        ...
+
+**The third row is mandatory in this kind of datasets***, in order to know which attributes correspond to the users (row=1) and which ones to the items (col=1). For the case of big datasets, users and items must be specified as continuous attributes due to efficiency issues. (*Note: If the meta attributes _row_ or _col_, some simple heuristics will be applied: users=_column 0_, items=_column 1_, class=_last column_)
+
+Here are the first few lines from a data set :
+
+    
+        user            movie         score         tid
+        continuous      continuous    continuous    time
+        row=1           col=1         class         meta
+        196             242           3             881250949
+        186             302           3             891717742
+        22              377           1             878887116
+        244             51            2             880606923
+        166             346           1             886397596
+        298             474           4             884182806
+        ...
+    
+
+
+
+
+### 
+
+
+
+
+### Training a model
+
+
+This step is pretty simple. To train a model we have to load the data as is described above and connect it to the learner. (_Don't forget to click apply_)
+
+{{% figure src="/images/2016/08/data-to-brismf.png" width="300" %}}
```

Comment:
```suggestion
<Figure src="/images/2016/08/data-to-brismf.png" width="300" />
```

+ there are some instances, that start with `{{< figure` and end with `>}}`

Reply:
What about the following instances?

```
{{< window-screenshot src="/blog_img/2022/...
{{% window-screenshot src="/blog_img/2022/...
{{< figure src="/blog_img/2022/...
{{% video src="/blog_img/2022/...
{{< workflow-screenshot src="/blog_img/2022/...
{{% workflow-screenshot src="/blog_img/2022/...
```

Reply:
I will do those with find/replace.